real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/GuiUtil.java

```diff
@@ -0,0 +1,89 @@
+package com.roboclub.robobuggy.ui;
+
+import java.awt.Component;
+import java.awt.Container;
+
+import com.roboclub.robobuggy.main.RobobuggyMessageLevel;
+import com.roboclub.robobuggy.main.RobobuggyLogicException;
+
+/**
+ * This is a helper class for the gui containing commonly used gui related helper function
+ * @author Trevor Decker
+ *
+ */
+public class GuiUtil {
+	
+	//This functions splits the current gui pane into the given number of rows and columns
+	public static void subPlot(int rows, int cols, int startRow, int startCol, int stopRow, int stopCol,Component comp,Container pane){
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method subPlot() has an NPath complexity of 1024](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1801801910/issues/source?bid=2938665&fileBranchId=3059924#l17)